### ISSUE TITLE
GET /config should return a JSON instead of a stringified YAML

### ISSUE THREAD

**Issue by hbagdi** [Kong/kong#5927]
### Summary

`GET /config` endpoint on the Admin API is primarily intended for debugging purposes to figure out the current running configuration of Kong.

Currently, it returns a YAML encoded as a string in the response:

```
http :8001/config                  
HTTP/1.1 200 OK
Access-Control-Allow-Origin: *
Connection: keep-alive
Content-Length: 662
Content-Type: application/json; charset=utf-8
Date: Tue, 19 May 2020 20:01:18 GMT
Server: kong/2.0.4
X-Kong-Admin-Latency: 120

{
    "config": "_format_version: '1.1'\nservices:\n- host: mockbin.org\n  created_at: 1589918471\n  connect_timeout: 60000\n  id: 2254ba65-245b-5b72-a0b8-7341d45f5e77\n  protocol: http\n  name: svc1\n  read_timeout: 60000\n  port: 80\n  updated_at: 1589918471\n  tags:\n  - team-svc1\n  write_timeout: 60000\n  retries: 5\nroutes:\n- created_at: 1589918471\n  id: 4726a05a-d550-537e-9fc3-a07c7a1188c7\n  service: 2254ba65-245b-5b72-a0b8-7341d45f5e77\n  name: r1\n  updated_at: 1589918471\n  preserve_host: false\n  regex_priority: 0\n  strip_path: true\n  paths:\n  - /r1\n  https_redirect_status_code: 426\n  protocols:\n  - http\n  - https\n  path_handling: v0\n"
}
```

This is unreadable and difficult to parse correctly for large configurations.

The endpoint should instead return a JSON-encoded document instead.
An example:

```
http :8001/config
HTTP/1.1 200 OK
Access-Control-Allow-Origin: *
Connection: keep-alive
Content-Length: 681
Content-Type: application/json; charset=utf-8
Date: Tue, 19 May 2020 20:01:11 GMT
Server: kong/2.0.4
X-Kong-Admin-Latency: 217

{
   "_format_version": "1.1",
    "routes": {
        "4726a05a-d550-537e-9fc3-a07c7a1188c7": {
            "created_at": 1589918471,
            "https_redirect_status_code": 426,
            "id": "4726a05a-d550-537e-9fc3-a07c7a1188c7",
            "name": "r1",
            "path_handling": "v0",
            "paths": [
                "/r1"
            ],
            "preserve_host": false,
            "protocols": [
                "http",
                "https"
            ],
            "regex_priority": 0,
            "service": {
                "id": "2254ba65-245b-5b72-a0b8-7341d45f5e77"
            },
            "strip_path": true,
            "updated_at": 1589918471
        }
    },
    "services": {
        "2254ba65-245b-5b72-a0b8-7341d45f5e77": {
            "connect_timeout": 60000,
            "created_at": 1589918471,
            "host": "mockbin.org",
            "id": "2254ba65-245b-5b72-a0b8-7341d45f5e77",
            "name": "svc1",
            "port": 80,
            "protocol": "http",
            "read_timeout": 60000,
            "retries": 5,
            "tags": [
                "team-svc1"
            ],
            "updated_at": 1589918471,
            "write_timeout": 60000
        }
    }
}

``` 

The format of the response should conform to the format specification of declarative configuration of Kong (https://discuss.konghq.com/t/rfc-kong-native-declarative-config-format/2719).

**Comment by bungle** [comment]
@hbagdi funny that you are asking it, as it looks like you implemented that in a first place: https://github.com/Kong/kong/pull/4971

So do you consider this as a bug?

**Comment by bungle** [comment]
@hbagdi ah I see, you mean it is time to implement:

> Once JSON is supported as an output format, it can be added to this
endpoint in a non-breaking change fashion by looking up the
content-type in the accept header sent in the request.

**Comment by hbagdi** [comment]
> funny that you are asking it, as it looks like you implemented that in a first place: #4971

Yeah, I tried attempting this myself before opening this issue but couldn't find a quick way to do it, and hence opened an issue.
